setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = lomap2
+description = Lead Optimization Mapper 2
+long_description = file: README.md
+long_description_type = text/markdown
+author = Gaetano Calabro and David Mobley
```

Comment:
Add Toni here? She had been listed as a maintainer previously, but now we've taken that.

Reply:
I think those two are still the authors, then there's a bunch of contributors/maintainers, it's tricky